ArduCopter/mode_flip.cpp

```diff
@@ -35,16 +35,13 @@
 // flip_init - initialise flip controller
 bool ModeFlip::init(bool ignore_checks)
 {
-    // only allow flip from ACRO, Stabilize, AltHold or Drift flight modes
-    if (copter.control_mode != Mode::Number::ACRO &&
-        copter.control_mode != Mode::Number::STABILIZE &&
-        copter.control_mode != Mode::Number::ALT_HOLD &&
-        copter.control_mode != Mode::Number::FLOWHOLD) {
+    // only allow flip from ACRO, Stabilize, AltHold or FlowHold flight modes
```

Comment:
nitpick: Should make this comment generic - "some flight modes, for example ...".

Reply:
Thank you, fixed it.